make a sin method in java (#1311)
Codecademy/docs: content/java/concepts/math-methods/terms/sin/sin.md

---
Title: '.sin()'
Description: 'Returns the trigonometric sine of the specified angle.'
Subjects:
  - 'Computer Science'
Tags:
  - 'Arithmetic'
  - 'Functions'
  - 'Methods'
  - 'Numbers'
CatalogContent:
  - 'learn-java'
  - 'paths/computer-science'
---

> The **`Math.sin()`** method returns the sine of an angle argument, given in radians. The result is in the range [-1,1].

Reviewer: ```suggestion
The **`Math.sin()`** method returns the trigonometric sine of an angle argument, given in radians.
```

## Syntax

```pseudo
Math.sin(angle)
```
The `angle` parameter is expressed in radians.

>    - returns the `trigonometric` sine of the specified angle

Reviewer 2: ```suggestion
- Returns the `trigonometric` sine of the specified angle.
```

> The `angle` parameter is expressed in radians.
> 
> - Returns the `trigonometric` sine of the specified angle.
> - Returns `NaN` if the specified angle is NaN or infinity.

Reviewer: ```suggestion

The `angle` parameter is expressed in radians.

- The result is in the range [-1,1].
- If the `angle` is `NaN` or infinity, `NaN` is returned.
```

>   -  returns `NaN` if the specified angle is NaN or infinity

Reviewer 2: ```suggestion
- Returns `NaN` if the specified angle is NaN or infinity.
```

## Example

The following example demonstrates the application of `.sin()` method:

```java
// Check.java
public class Check {
  public static void main(String args[]) {
  // convert degrees to radians
  double pi = Math.PI;
  double degree = 60;
  double radian = degree * pi/180;
  System.out.println( "Sine of 60 degrees is " + Math.sin(radian));
  }
}
```

This results in the following output:

> ```java

Reviewer: ```suggestion
```shell
```

---
Title: '.sin()'
Description: 'Returns the trigonometric sine of the specified angle.'
Subjects:
  - 'Computer Science'
Tags:
  - 'Arithmetic'
  - 'Functions'
  - 'Methods'
  - 'Numbers'
CatalogContent:
  - 'learn-java'
  - 'paths/computer-science'
---

The **`Math.sin()`** method returns the trigonometric sine of an angle argument, given in radians.

## Syntax

```pseudo
Math.sin(angle)
```

The `angle` parameter is expressed in radians.

- The result is in the range [-1,1].
- If the `angle` is `NaN` or infinity, `NaN` is returned.

## Example

The following example demonstrates the application of `.sin()` method:

```java
// Check.java

> public class Check {
>   public static void main(String args[]) {
>   // convert degrees to radians
>   double pi = Math.PI;
>   double degree = 60;
>   double radian = degree * pi/180;
>   System.out.println( "Sine of 60 degrees is " + Math.sin(radian));
>   }
> }

Reviewer: ```suggestion
public class Check {
  public static void main(String args[]) {
    // convert degrees to radians
    double pi = Math.PI;
    double degree = 60;
    double radian = degree * pi/180;
    System.out.println( "Sine of 60 degrees is " + Math.sin(radian));
  }
}
```

---
Title: '.sin()'
Description: 'Returns the trigonometric sine of the specified angle.'
Subjects:
  - 'Computer Science'
Tags:
  - 'Arithmetic'
  - 'Functions'
  - 'Methods'
  - 'Numbers'
CatalogContent:
  - 'learn-java'
  - 'paths/computer-science'
---

The **`Math.sin()`** method returns the sine of an angle argument, given in radians. The result is in the range [-1,1].

## Syntax

```pseudo
Math.sin(angle)
```
The `angle` parameter is expressed in radians.

> 

Reviewer 2: ```suggestion
```

   - returns the `trigonometric` sine of the specified angle
  -  returns `NaN` if the specified angle is NaN or infinity

> 

Reviewer 2: ```suggestion
```

---
Title: '.sin()'
Description: 'Returns the trigonometric sine of the specified angle.'
Subjects:
  - 'Computer Science'
Tags:
  - 'Arithmetic'
  - 'Functions'
  - 'Methods'
  - 'Numbers'
CatalogContent:
  - 'learn-java'
  - 'paths/computer-science'
---

> The **`Math.sin()`** method returns the sine of an angle argument, given in radians, in the range [-1,1].

Reviewer 2: ```suggestion
The **`Math.sin()`** method returns the sine of an angle argument, given in radians. The result is in the range [-1,1].
```

## Syntax

```pseudo
Math.sin(angle)
```
The `angle` parameter is expressed in radians.

> ## sin() Return Value

Reviewer 2: Remove this header so the below is part of the Syntax section.
